ga4gh/cli.py

```diff
@@ -9,12 +9,68 @@
 
 import time
 import argparse
+import sys
 
 import ga4gh.frontend as frontend
 import ga4gh.client as client
 import ga4gh.backend as backend
 import ga4gh.protocol as protocol
 import ga4gh.datamodel.variants as variants
+import ga4gh.converters as converters
+
+
+##############################################################################
+# ga2vcf
+##############################################################################
+
+
+def ga2vcf_main(parser=None):
+    if parser is None:
+        parser = argparse.ArgumentParser(
+            description="GA4GH VCF conversion tool")
+    addClientGlobalOptions(parser)
+    subparsers = parser.add_subparsers(title='subcommands',)
+    addVariantSetsSearchParser(subparsers)
+    args = parser.parse_args()
+    request = protocol.GASearchVariantSetsRequest()
```

Comment:
This should be `SearchVariants` not `SearchVariantSets` because we want to be able to specify all of the range options available for `SearchVariants`. We will probably never want to convert a whole variant set, we'll just want to convert some variants for some small range we're interested in.